diff -p -up hsfmodem-7.80.02.03full/scripts/cnxtconfig.in.orig hsfmodem-7.80.02.03full/scripts/cnxtconfig.in
--- hsfmodem-7.80.02.03full/scripts/cnxtconfig.in.orig	2009-02-20 18:30:17.000000000 -0300
+++ hsfmodem-7.80.02.03full/scripts/cnxtconfig.in	2009-04-16 15:41:31.000000000 -0300
@@ -2972,7 +2972,7 @@ fi
 
 # If nothing explicitely specified, configure everything
 if ! ${explicitopt}; then
-	do_cfgkernel=true
+#	do_cfgkernel=true
 	do_cfgserial=true
 	if [ "@CNXTDRIVER@" != "dgc" ]; then
 	do_cfgregion=true
